uhdm-integration/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X.X', GIT_DESCRIBE_NUMBER|int,   GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: uhdm-integration
+  version: {{ version }}
+
+source:
+  git_rev: master
+  git_url: https://github.com/antmicro/uhdm-integration.git
+
+build:
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - python
+    - pkg-config
+    - libuuid
+    - gperftools
+    - flex
+    - libunwind
+    - swig
+  host:
+    - python
+  run:
+    - {{ compiler('cxx') }}
+    - readline ==8.0
+  {% for package in resolved_packages('build') %}
+    - {{ package }}
+  {% endfor %}
+
+outputs:
+    - name: uhdm-integration-verilator
+      script: verilator-flow.sh
+      test:
+          commands:
+              - uhdm-surelog --version
+              - uhdm-verilator --version
+    - name: uhdm-integration-yosys
+      script: yosys-flow.sh
+      test:
+          commands:
+              - uhdm-surelog --version
+              - uhdm-yosys --version
```

Comment:
Could we create separate packages for yosys and verilator? For now maybe it makes sense to keep them together, but in the long run it makes sense for them not to be packed together.

Additionally it will be much easier to integrate them with sv-tests if they are separated.

Reply:
Ok, nvm, I wasn't aware you can generate two packages from a single build scripts like you do here!